tests/test_example_objectscorelib.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+import boto3
+from moto import mock_s3
+from omegaconf import OmegaConf
+
+from examples.objects_core_lib.core_lib.objects_core_lib import ObjectsCoreLib
+
+
+class ObjectsCoreLibTesting(ObjectsCoreLib):

```

Comment:
what is the purpose of this class?

Reply:
for passing config to `object_core_lib` for now the current config will require docker for env variables, I tried to edit the config but boto3 fails at that time, so I made this class to pass a basic config file to `object_core_lib`

Reply:
please use ObjectsCoreLib directly